src/main/java/core/basesyntax/service/impl/CalculationServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.CalculationService;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class CalculationServiceImpl implements CalculationService {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+
+    @Override
+    public void initializationStorage(List<String> convertedFileIntoList) {
+        List<String[]> separatedData = separateData(convertedFileIntoList);
+        Map<String, Integer> startBalance = separatedData.stream()
+                .filter(i -> i[OPERATION_INDEX].equals(Operation.BALANCE.getCode()))
+                .collect(Collectors.toMap(i -> i[FRUIT_INDEX],
+                        i -> Integer.parseInt(i[AMOUNT_INDEX])));
+        Storage.setStorage(startBalance);
+    }
+
+    @Override
+    public void calculation(List<String> convertedFileIntoList) {
```

Comment:
You need to realize the pattern `Strategy`. That means you need to have an interface with one method and 4 realizations for every single operation (b, p, s, r). Also, you need to have an additional class^ that will return as a necessary realization by operation. 

Don't forget to separate services and strategies in different packages